pkg/didcomm/transport/http/inbound.go

```diff
@@ -121,6 +128,8 @@ func (i *Inbound) Start(prov transport.InboundProvider) error {
 		return fmt.Errorf("HTTP server start failed: %w", err)
 	}
 
+	i.server = &http.Server{Addr: ":" + i.port}
```

Comment:
Any specific reason to move the server initiation from New() to Start() function ? If Start() is called multiple times, we might have multiple servers created for same instance.

Reply:
Not sure why I did that originally, but you're right. I moved it back to the New function.